infrastructure/front-door-docs.tf

```diff
@@ -1,22 +1,23 @@
-resource "azurerm_cdn_frontdoor_profile" "docs" {
-  name                = "${local.org}-fd-${local.service_name}-docs-${var.environment}"
-  resource_group_name = azurerm_resource_group.primary.name
-  sku_name            = "Standard_AzureFrontDoor"
+# resource "azurerm_cdn_frontdoor_profile" "docs" {
+#   name                = "${local.org}-fd-${local.service_name}-docs-${var.environment}"
+#   resource_group_name = azurerm_resource_group.primary.name
+#   sku_name            = "Standard_AzureFrontDoor"
 
-  tags = local.tags
-}
+#   tags = local.tags
+# }
```

Comment:
No need to comment out, let's just delete - they aren't coming back! Same in the web file.